src/Umbraco.Cloud.StorageProviders.AzureBlob/AzureBlobOptions.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Umbraco.Cloud.StorageProviders.AzureBlob
+{
+    /// <summary>
+    /// The Azure Blob options used on Umbraco Cloud.
+    /// </summary>
+    internal class AzureBlobOptions
+    {
+        /// <summary>
+        /// Gets or sets the endpoint.
+        /// </summary>
+        /// <value>
+        /// The endpoint.
+        /// </value>
+        [Required]
+        public string Endpoint { get; set; } = null!;
+
+        /// <summary>
+        /// Gets or sets the shared access signature.
+        /// </summary>
+        /// <value>
+        /// The shared access signature.
+        /// </value>
+        [Required]
+        public string SharedAccessSignature { get; set; } = null!;
+
+        /// <summary>
+        /// Gets the connection string.
+        /// </summary>
+        /// <value>
+        /// The connection string.
+        /// </value>
+        public string ConnectionString => $"BlobEndpoint={Endpoint};SharedAccessSignature={SharedAccessSignature}";
```

Comment:
If the property is renamed, this should also be updated:
```suggestion
        public string ConnectionString => $"BlobEndpoint={BlobEndpoint};SharedAccessSignature={SharedAccessSignature}";
```